### PR TITLE
Bugfixes from #387

### PR DESCRIPTION
**Pull request by mjashanks**
bugfixes from #387 

see indivual commits for details

### CODE REVIEW

File: packages/builder/rollup.config.js [hunk 1]
```diff
@@ -185,7 +185,11 @@ export default {
     svelte({
       // enable run-time checks when not in production
       dev: !production,
-      include: ["src/**/*.svelte", "node_modules/**/*.svelte"],
+      include: [
+        "src/**/*.svelte",
+        "node_modules/**/*.svelte",
+        "../../../bbui/src/**/*.svelte",
```
Comment (mjashanks) [inline packages/builder/rollup.config.js:191]: I had to do this in order for me `yarn link` to bbui to work. It assumes that bbui and budibase are in the same parent directory.

File: packages/builder/src/pages/[application]/_reset.svelte [hunk 2]
```diff
@@ -27,6 +27,17 @@
       throw new Error(pkg)
     }
   }
+
+  const topItemNavigate = path => () => {
```
Comment (shogunpurple) [inline packages/builder/src/pages/[application]/_reset.svelte:31]: What was happening here?

Comment (mjashanks) [reply packages/builder/src/pages/[application]/_reset.svelte:36]: ah yes - iv added some comments

Comment (shogunpurple) [reply packages/builder/src/pages/[application]/_reset.svelte:36]: cool, makes sense -  I wonder if routify has some kind of first-class API for this? Couldn't see anything in the docs

**Review by shogunpurple — APPROVED** [review]
LGTM 🚀

**Pull request merged**